typings/dart.d.ts

```diff
@@ -26,3 +26,5 @@ export enum Order {
 }
 
 export declare const dartGenerator: any;
+
+export {DartGenerator} from './generators/dart';
```

Comment:
If they are identical (which they should be), you might as well delete the handwritten `Order` declaration and use the official one: 

```suggestion
export {Order, DartGenerator} from './generators/dart';
```

(here and for the other four).

Reply:
I think there's some small value in not doing this, a) if I've done something wrong with the generator class, at least I'm not going to break anyone in this patch release because that wasn't previously exported. I don't really want to find out there is a material difference in the Order enum in the patch release, I'd rather wait to find that out through beta feedback (hopefully) and b) I'm trying to create minimal merge conflicts between this and the rc/v11 branch, because I find dealing with merge conflicts when you're trying to perform a release can be a bit harrowing. That's also why I opted not to adapt the tests from #7750.

But, I think this value is somewhat small, so if you disagree, I can add this in. I'll leave this PR open for now because Monday is the earliest we should do a patch release anyway.

Reply:
I think that's fine.  I also think we should roll-up `develop` into the v11 branch before the next beta release (which will follow shortly once #7822 has landed) and I'm happy to take care of that merge and any conflicts even if I'm not the one doing the release.